Jenkinsfile

```diff
@@ -1,150 +1,140 @@
 max_time = 180
 
+setup_pip_venv = """
+    rm -rf venv
+    conda list
+
+    python3 -m venv venv
+    source venv/bin/activate
+    python3 -m pip install -U pip setuptools wheel
```

Comment:
might need to separate the install pip line with the others, since pip version dictates the way setuptools and wheel are installed:

```
python3 -m pip install -U pip
python3 -m pip install -U setuptools wheel
```

Reply:
good point -- updated to be on a separate line